### PR TITLE
Create OpenAPI Deploy docs github action

### PR DESCRIPTION
**Pull request by albandum**
## Description

Create a github action to deploy the swagger.json docs to Readme.io
For now push is manual through Github action run.

### CODE REVIEW

**Review by lasryaric — APPROVED** [review]
LGTM 🔥

File: .github/workflows/deploy-docs-openapi.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# This GitHub Actions workflow was auto-generated by the `rdme` cli on 2024-06-25T13:33:11.644Z
+# You can view our full documentation here: https://docs.readme.com/docs/rdme
+name: Deploy OpenAPI Docs
```
Comment (flvndvd) [inline .github/workflows/deploy-docs-openapi.yml:3]: Do we need to run it every time, or can we limit it to only the subset of files that have changed?

Comment (albandum) [reply .github/workflows/deploy-docs-openapi.yml:3]: Whenever a JSDoc comment is changed - the openAPI doc is not generated from the code itself but from the JSDoc comments

Comment (flvndvd) [reply .github/workflows/deploy-docs-openapi.yml:3]: I understand, but we're only concerned with the public API, correct? So, we should focus on all */v1/* paths and not the others, or am I missing something?

Comment (albandum) [reply .github/workflows/deploy-docs-openapi.yml:3]: Not sure to understand what you mean - It's basically publishing whatever is present in `public/swagger.json` (the workflow itself doesn't know about our architecture). 

So yes, we only need to add JSDoc in the v1/ files - I'm adding a rule [in this PR](https://github.com/dust-tt/dust/pull/5867) to make sure all /v1 routes are covered

Comment (flvndvd) [reply .github/workflows/deploy-docs-openapi.yml:3]: Seen IRL, this won't be triggered automatically.

**Pull request merged**